src/scss/partial/_footer.scss

```
@@ -0,0 +1,132 @@
+.footer{
+    @include w-h(100%, 100%);
+    .footer__top{
+        @include w-h(100%, 100%);
+        border-top: 2px solid $black;
+        &-brand{
+            background-image: url(../../assets/img/brand-type-film-footer_ver2.png);
+            background-size: cover;
+            @include w-h(100%, 2rem);
+        }
+    }
+    .footer__middle{
+        border-top: 2px solid $black;
+        &__policy{
+            padding: .5rem 0 1rem 0;
+            @include displayFlex(space-between, none);
+            .policy--item{
```

```suggestion
    .footer__middle{
        border-top: 2px solid $black;
        
        &__policy{
            @include displayFlex(space-between, none);
            padding: .5rem 0 1rem 0;
            
            .policy--item{
```
Mỗi block cách nhau 1 dòng nhé